src/com/jambit/onboarding2020/tbrpg/domain/Room/HangmanRoom.java

```diff
@@ -0,0 +1,120 @@
+package com.jambit.onboarding2020.tbrpg.domain.Room;
+
+import com.jambit.onboarding2020.tbrpg.core.GameInput;
+import com.jambit.onboarding2020.tbrpg.core.InvalidInputException;
+import com.jambit.onboarding2020.tbrpg.domain.Player.Player;
+import com.jambit.onboarding2020.tbrpg.domain.Player.PlayerDeadException;
+import com.jambit.onboarding2020.tbrpg.games.hangman.Hangman;
+import com.jambit.onboarding2020.tbrpg.games.hangman.HangmanLoseException;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.InputStreamReader;
+import java.util.Arrays;
+import java.util.Random;
+import java.util.Scanner;
+
+public class HangmanRoom extends AbstractRoom {
+
+    private final String wordlist = System.getProperty("user.dir") + "/resources/nouns_10-20.txt";
+
+    @Override
+    public void printWelcomeMessage() {
+        System.out.println("Willkommen zu " +
+                "\n _" +
+                "\n| |" +
+                "\n| |__   __ _ _ __   __ _ _ __ ___   __ _ _ __  " +
+                "\n| '_ \\ / _` | '_ \\ / _` | '_ ` _ \\ / _` | '_ \\ " +
+                "\n| | | | (_| | | | | (_| | | | | | | (_| | | | |" +
+                "\n|_| |_|\\__,_|_| |_|\\__, |_| |_| |_|\\__,_|_| |_|" +
+                "\n                    __/ |" +
+                "\n                   |___/" +
+                "\n"
+        );
+    }
+
+    @Override
+    public void enter() throws PlayerDeadException {
+
+        GameInput in = new GameInput(new InputStreamReader(System.in));
+        String line = "";
+        String randomWord = "TESTWORD";
+
+        try {
+            randomWord = this.getRandomWord();
+        } catch (FileNotFoundException e) {
+            e.printStackTrace();
+        }
+
+        Hangman hangman = new Hangman(randomWord);
+        hangman.drawPitch();
+        hangman.drawLetterInput();
+
+        while (in.gameState()) {
```

Comment:
I recommend to rename the method gameState(), since I have no glue what it should mean. 😄